geode-core/src/main/java/org/apache/geode/management/internal/cli/domain/StackTracesPerMember.java

```diff
@@ -19,16 +19,16 @@
 public class StackTracesPerMember implements Serializable {
 
   private static final long serialVersionUID = 1L;
-  private final String memberNameOrId;
+  private final String memberNameOrIdWithTimestamp;
```

Comment:
Can we split these concepts (`memberId` and `timestamp`) in two different attributes?.
I don't see any good reason about why they should just be one single attributed created by a String concatenation (plus the `at` word). This logic is basically part of the "view" or representation and, as such, it should be handled directly in the gfsh command.